### ISSUE TITLE
should be devDeps warnings for common dev deps (eslint* depcheck, jest, etc)

### ISSUE THREAD

**Issue by twk-b** [depcheck/depcheck#435]
### Feature request description

some packages should almost never be in dependencies, only in devDependencies:
examples:

* depcheck
* eslint
* eslint-plugin-node

### Code snippets (if applicable)
```
depcheck --checkDevDep
```

package.json
```
{
   "dependencies": " { "eslint-plugin-node": "1.0.0" }
   "devDependencies": " { "eslint": "1.0.0", "depcheck": "1.0.0"  }
}
```

output:
warn: eslint-plugin-node: is likely a devDepencendies

**Comment by VincentLanglet** [comment]
I don't see this like a good feature because the 'should be devDependency' list may be subjective.

> some packages should almost never be in dependencies
You said `almost`. False warning is bad.
